src/main/java/com/synopsys/integration/alert/provider/blackduck/descriptor/BlackDuckProviderUIConfig.java

```diff
@@ -31,22 +31,18 @@
 import com.synopsys.integration.alert.common.descriptor.config.field.NumberConfigField;
 import com.synopsys.integration.alert.common.descriptor.config.field.PasswordConfigField;
 import com.synopsys.integration.alert.common.descriptor.config.field.ReadOnlyConfigField;
-import com.synopsys.integration.alert.common.descriptor.config.ui.DescriptorMetadata;
 import com.synopsys.integration.alert.common.descriptor.config.ui.UIConfig;
-import com.synopsys.integration.alert.common.enumeration.ConfigContextEnum;
-import com.synopsys.integration.alert.common.enumeration.DescriptorType;
-import com.synopsys.integration.alert.provider.blackduck.BlackDuckProvider;
 
 @Component
 public class BlackDuckProviderUIConfig extends UIConfig {
     private static final String PROXY_SUB_GROUP = "Proxy Configuration";
 
-    @Override
-    public DescriptorMetadata generateDescriptorMetadata() {
-        return new DescriptorMetadata("Black Duck", "blackduck", BlackDuckProvider.COMPONENT_NAME, DescriptorType.PROVIDER, ConfigContextEnum.GLOBAL, "laptop", setupFields());
+    public BlackDuckProviderUIConfig() {
+        super("Black Duck", "blackduck", "laptop");
```

Comment:
Encapsulate lowercase blackduck and laptop strings into static final strings in the descriptor class since they are the URL and font-awesome icon name.